src/app/app.js

```diff
@@ -656,6 +633,58 @@ if (nw.App.fullArgv.indexOf('-f') !== -1) {
     win.enterFullscreen();
 }
 
+// nwjs window events
+win.on('resize', function (width, height) {
+    localStorage.width = Math.round(width);
+    localStorage.height = Math.round(height);
+});
+
+win.on('move', function (x, y) {
+    localStorage.posX = Math.round(x);
+    localStorage.posY = Math.round(y);
+});
+
+win.on('enter-fullscreen', function () {
+    App.vent.trigger('window:focus');
+});
+
+win.on('minimize', function () {
+    if (Settings.minimizeToTray) {
+        minimizeToTray();
+    }
+});
+
+win.on('maximize', function () {
+    localStorage.isMaximized = 1;
+    if (process.platform === 'win32') {
+        $('.os-max').addClass('os-is-max');
```

Comment:
because other use the dot thingy, at least that's what I remember. Maybe os-is-max won't hurt them, and we can set it anyway. Idk